SPEC.md

```diff
@@ -198,7 +194,9 @@ linespace := newline | ws | single-line-comment
 
 newline := `000D` | `000A` | `000D` `000A` | `0085` | `000C` | `2028` | `2029`
 
-ws := unicode-space | multi-line-comment
+ws := bom |unicode-space | multi-line-comment
+
+bom := `FFEF`
```

Comment:
I know we’re already using it elsewhere, but I’m not terribly fond of backticks to denote unicode scalars. I’d rather we use `U+FFEF` instead, which is the same number of characters anyway.

Reply:
Yeah let's fix that (I'm going to bed for the night so tomorrow?